scripts/wad-diff

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
If it is at all possible, change to use `/bin/sh` (and use shellcheck to verify syntax). I won't consider it bad to require bash, however.